Lib/pathlib/_abc.py

```diff
@@ -846,6 +846,11 @@ def copytree(self, target, *, follow_symlinks=True,
         if on_error is None:
             def on_error(err):
                 raise err
+        if target.is_relative_to(self):
+            try:
+                raise OSError(f"Cannot copy {self!r} inside itself: {target!r}")
+            except OSError as err:
+                on_error(err)
```

Comment:
Don't you need a return here? or something else? Otherwise, the flow continues normally.

Reply:
Yes indeed, thanks.